docs/source/models/concept_models/vivarium_census_synthdata/concept_model.rst

```diff
@@ -2807,6 +2807,284 @@ have not been included. These are:
 #. We sample 100% of events. This is likely unrealistic, but the percent is probably very high still. 
 #. There are errors in SSN data, which are not modeled here. 
 
+Noise Functions
+^^^^^^^^^^^^^^^
+
+In order to have a realistic challenge with PRL, it is essential to have 
+noise added to the data. Noise is generally divided into two types: 
+
+#. **Column based noise:** errors in individual data entry, such as miswriting or incorrectly selecting responses 
+#. **Row based noise:** errors in the inclusion or exclusion of entire rows of data, such as duplication or omission 
+
+To begin, we will start with defining column based noise. Some general rules 
+for all column based noise include: 
+
+- The amount of each type of noise is individually configurable for each observer up to the max listed in the table below. This means that the end user can select that: census first names have 5% typographic errors. The minimum noise is 0 for all column noise. 
+- Simulants are selected for noise at random. Selection is not based on any attributes and simulants do not have a higher or lower propensity for noise that would carry across observers (e.g., there are not "messy" simulants who make errors on all forms). 
+- As noise functions for certain data types are common across observers, the table below is organized by data type (first name). Below the table, there is further information and defintion on each noise type. 
+
+.. list-table:: Type of Noise Included and Maximum Level by Data Included
+  :widths: 20 20 20 20 20 
+  :header-rows: 0
+
+  * - Data in Observer
+    - Observers Present 
+    - Maximum Noise Level 
+    - Types of Noise 
+    - Notes
+  * - First Name
+    - Census, Household Surveys, WIC, Taxes (both), SSA  
+    - 5%
```

Comment:
Just to confirm I am understanding: this means 5% for _each_ error type? So roughly 25% of people getting their name wrong?

This seems very high. We should pay more attention to these defaults but I don't think they are blocking, so maybe add a TODO that they are not final.

Reply:
Hahah well I was thinking of this as a max! Yes, that would be insane. When I thought of a max, it was more if someone wanted to test very high noise of a specific type to see how their PRL would handle it. If it is a default then yes, I will definitely turn this down a good amount.